gene/fromGtf.go

```diff
@@ -40,11 +40,9 @@ func GtfToGene(g *gtf.Gene, ref []*fasta.Fasta) *Gene {
 				for i = val.FiveUtr.Start - 1 - answer.startPos; i < val.FiveUtr.End-answer.startPos; i++ {
 					answer.featureArray[i] = UtrFive
 				}
-			}
-
-			if val.ThreeUtr != nil {
-				for i = val.ThreeUtr.Start - 1 - answer.startPos; i < val.ThreeUtr.End-answer.startPos; i++ {
-					answer.featureArray[i] = UtrThree
+				answer.cdnaSeq = append(answer.cdnaSeq, fastaMap[transcript.Chr][val.FiveUtr.Start-1:val.FiveUtr.End]...)
+				if val.FiveUtr.End-answer.startPos > answer.utrFive.end {
```

Comment:
I am not totally sure what is happening here.  Not saying it is a bug, just not sure I completely follow it.  I am thinking we are changing coordinate systems where "val" uses zero as the first position in the chromosome and answer uses zero as the first position of transcription.  I think a 1-line comment might help me.

Reply:
Correct, the UTR sequences are being recorded relative to the cDNA such that the first base of the 5' UTR is base zero. I added a comment.